timescale-forge/page-index/page-index.js

```diff
@@ -18,6 +18,10 @@ module.exports = [
       {
         title: "Create a Multi-node Cluster",
         href: "forge-multi-node"
+      },
+      {
+        title: "VPC Peering",
```

Comment:
Most of the others are with noun form, so maybe:

"Peer a VPC network" or something?

(Also, one of the earlier should probably be "Scale" vs. "Scaling"

Reply:
I do think that "VPC Peering" has the advantage of being colloquial (probably thanks to AWS), so it might be more easily identifiable to users.

Happy to go whichever way you prefer. Just let me know `:)`

Reply:
I'm going to normalise all these at some point in the future anyway, so don't stress too much right now. Generally speaking, though, it should be a noun phrase for a concept, and a verb phrase for a procedure. "VPC peering" is a noun phrase, while "Peer a VPC network" is a verb phrase, so the page itself should be called "VPC peering" and the procedures within it called things like "Peer a VPC network". 

Also, don't forget to use sentence case for titles: capitalise the first word, and proper nouns, only.

Reply:
```suggestion
        title: "VPC peering",
```